src/com/intellij/rt/coverage/util/OptionsUtil.java

```diff
@@ -22,6 +22,7 @@ public class OptionsUtil {
           && "true".equals(System.getProperty("idea.new.tracing.coverage", "true"));
   public static final boolean NEW_TEST_TRACKING_ENABLED = "true".equals(System.getProperty("idea.new.test.tracking.coverage", "true"));
   public static boolean CONDY_ENABLED = "true".equals(System.getProperty("coverage.condy.enable", "true"));
+  public static boolean INDY_ENABLED = "true".equals(System.getProperty("coverage.indy.enable", "true"));
```

Comment:
I think it worth setting this option to `false` by default for now

Reply:
Happy to have it off by default if you're tentative, but IMO the 1-time extra hash table lookup per method is very comparable in speed to the extra transformation per method that the field approach does.